stimuli-ci-generation/install.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
I don't see this script called anywhere (or I might have missed it). Is this relevant? Why do we need to install google cloud sdk on our machine?

Reply:
My guess would be the program needs gcp sdk installed to access the storage buckets

Reply:
Maybe? I suspect it's cruft.